inc/mqttlog.class.php

```diff
@@ -100,10 +100,36 @@ protected function saveMqttMessage($direction, $topicList, $msg) {
       $msg = $DB->escape($msg);
       foreach ($topicList as $topic) {
          $topic = $DB->escape($topic);
+
+         $chunks = explode('/', $topic, 4);
+         if ($chunks[0] == '' || !isset($chunks[3])) {
+            // avoid save invalid topic strings
```

Comment:
I think you should describe more explictly the messages you want to filter.

Reply:
done